bin/docker-image-tool.sh

```diff
@@ -118,11 +120,17 @@ function build {
   # else use builtin spark
   if [[ ! -d "$KYUUBI_ROOT/spark-binary" ]]; then
     mkdir "$KYUUBI_ROOT/spark-binary"
+  else
+    rm -rf "$KYUUBI_ROOT/spark-binary/*"
```

Comment:
why we need delete it, if it is not existed

Reply:
The logic is if this dir exists, it is considered that the spark was used last time.  The spark version that was used last time is saved. Delete spark first and do copy or nothing according to `-w` parameters.